packages/teams-js/src/public/meeting.ts

```diff
@@ -210,6 +210,39 @@ export namespace meeting {
     error?: SdkError;
   }
 
+  /**
+   * Interface for mic state change
+   *
+   * @beta
+   */
+  export interface MicState {
+    id: number;
+    /**
+     * Indicates the mute status of the mic.
+     */
+    isMicMuted: boolean;
+  }
+
+  export interface MeetingFailedResponse {
+    id: number;
```

Comment:
id should have a user facing comment. I assume that this id is unique among MeetingFailedResponses and monotonically increasing?